### PR TITLE
Adding disablement of batch norm

### PR DESCRIPTION
**Pull request by Benjamin-Etheredge**
# Description

Simple exploratory fix to a bug when using pytorch object detectors. Batch norm layers weren't correctly frozen causing attacks that leverage those models to modify batch norm layers (thus causing model's performance to change unexpectedly)

Fixes # (issue)
#2601

## Type of change

Please check all relevant options.

- [ ] Improvement (non-breaking)
- [X] Bug fix (non-breaking)
- [ ] New feature (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Testing

Please describe the tests that you ran to verify your changes. Consider listing any relevant details of your test configuration.

- [ ] Test A
- [ ] Test B

**Test Configuration**:
- OS
- Python version
- ART version or commit number
- TensorFlow / Keras / PyTorch / MXNet version

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes have been tested using both CPU and GPU devices

### CODE REVIEW

**Review by beat-buesser — APPROVED** [review]
Hi @Benjamin-Etheredge Thank you very much for your pull request. I have merge your commit/solution in a separate push to include updates for the affected unit tests.
